includes/dart-integrations/firebase-remote-config.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Firebase Remote Config
+description: "Learn more about the Sentry Firebase Remote Config integration for the Dart SDK."
+sidebar_order: 6
+platforms:
+  - flutter
+---
+
+The `sentry_firebase_remote_config` integration provides [Firebase Remote Config](https://firebase.google.com/docs/remote-config/get-started?platform=flutter) support for Sentry thus providing insight into feature flag evaluations.
+
+## Behavior
+
+- Adding the `SentryFirebaseRemoteConfig` integration will automatically track feature flag evaluations for boolean Firebase Remote Config values.
+- Only boolean values are supported.
+
+## Prerequisites
+
+1. [Sentry SDK](/platforms/dart/#configure) version `9.0.0` or higher is installed.
+2. Firebase Remote Config is set up.
+
+## Install
+
+To use the `SentryFirebaseRemoteConfig` integration, add the `sentry_firebase_remote_config` dependency.
+
+```yml {filename:pubspec.yaml}
+dependencies:
+  sentry: ^9.0.0
+  sentry_firebase_remote_config: ^9.0.0
+```
+
+## Configure
+
+Add the `SentryFirebaseRemoteConfig` integration to the Sentry SDK initialization.
+
+```dart
+import 'package:sentry_flutter/sentry_flutter.dart';
+import 'package:sentry_firebase_remote_config/sentry_firebase_remote_config.dart';
+
+... // Initialize Firebase Remote Config
+
+await SentryFlutter.init(
+  (options) {
+    options.dsn = 'https://example@sentry.io/example';
+    options.addIntegration(
+      SentryFirebaseRemoteConfigIntegration(
+        firebaseRemoteConfig: firebaseRemoteConfig,
+        // Don't call `await remoteConfig.activate();` when firebase config is updated. Per default this is true.
```

Comment:
You're telling me not to call `await remoteConfig.activate()` depending on the value of the config below?
if so, I shouldn't call when the value is set to `false`? But the default is `true`?

Might be just me but this is a bit confusing

Reply:
I believe what it means is that they don't have to call `await remoteConfig.activate()` manually since the SDK does it already and that you can override this behaviour if you set `activateOnConfigUpdated: false`

Yea sounds confusnig, let's update this, maybe we can explain this in another section below